src/content/docs/codestream/codestream-integrations/msteams-integration.mdx

```diff
@@ -3,19 +3,21 @@ title: Microsoft Teams and CodeStream
 description: "How to connect CodeStream to your Microsoft Teams channel."
 ---
 
+Connect New Relic CodeStream to your Microsoft Teams channel.
+
 When you post a codemark your teammates will get notified via the activity feed,
 and potentially via email. Sometimes, though, you might want to share to
 Microsoft Teams as well. This would allow you to reach people who haven’t yet
 joined CodeStream, or maybe don’t spend a lot of time in their IDE. 
 
-## Connecting to Microsoft Teams
+## Connect to Microsoft Teams [#connect]
 
 1. [Click here to install the CodeStream
    app](https://teams.microsoft.com/l/app/7cf49ab7-8b65-4407-b494-f02b525eef2b?source=store-copy-link)
-   or go to "Apps" in your Teams sidebar and search for CodeStream.
+   or go to **Apps** in your Teams sidebar and search for CodeStream.
```

Comment:
do we capitalize this? My instinct is to treat it the same way we would treat a slack channel.

Reply:
It's tricky, because I don't have this software. But I'm coming around to your POV on this.